docs/json.md

```diff
@@ -21,6 +21,7 @@ In this chapter, we'll walk through features of [JSON](https://www.json.org/json
   * [Allowing special floating-point values](#allowing-special-floating-point-values)
   * [Class discriminator for polymorphism](#class-discriminator-for-polymorphism)
   * [Class discriminator output mode](#class-discriminator-output-mode)
+  * [Base64](#base64)
```

Comment:
Is there any particular reason for you to decide that it should be placed here in the list of features? Usually we add new entries to the end of the list, i.e. after 'Global naming strategy' in this case.

Reply:
Followed your advice from last [attempt](https://github.com/Kotlin/kotlinx.serialization/pull/1676#discussion_r725494382)

Reply:
I see.

> after 'class discriminator' section

And that was the last section in that list at the time :) So I would kindly ask you to move it a little bit down to the end of 'Json configuration' feature list. It is not ideal (since it is not actually related to `Json {}` config), but I can't think right now of a better place